CHANGES_NEXT_RELEASE

```diff
@@ -1,5 +1,4 @@
-- FIX: try get group by type using current type if no type is provided (#1155)
-- ADD: allow update polling device field (iota-json#602)
```

Comment:
Maybe these lines correspond to a wrong solved merge conflict and should be recovered?

Reply:
Fixed 2b008d2